### PR TITLE
Autohealingfixes

### PR DESCRIPTION
**Pull request by puneetg1983**
Fixing AutoHeal detector and ensuring time range selection

### CODE REVIEW

**Comment by stevenernst131** [comment]
You're going to need to wait on Shekhar's change and then resolve any merge conflicts.

**Comment by puneetg1983** [comment]
Gotcha!

File: AngularApp/src/app/availability/detector-view/detectors/autohealing-detector/autohealing-detector.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { AppAnalysisService } from '../../../../shared/services/appanalysis.service';
+import { DetectorControlService } from 'applens-diagnostics';
+
+@Component({
+  selector: 'autohealing-detector',
+  templateUrl: './autohealing-detector.component.html',
+  styleUrls: ['./autohealing-detector.component.css']
+})
+
+export class AutohealingDetectorComponent extends DetectorViewBaseComponent implements OnInit {
```
Comment (stevenernst131) [inline AngularApp/src/app/availability/detector-view/detectors/autohealing-detector/autohealing-detector.component.ts]: Are you still using this?

Comment (puneetg1983) [reply AngularApp/src/app/availability/detector-view/detectors/autohealing-detector/autohealing-detector.component.ts]: Hey Steve, not sure what you mean. I am using the AutohealingDetectorComponent

Comment (stevenernst131) [reply AngularApp/src/app/availability/detector-view/detectors/autohealing-detector/autohealing-detector.component.ts]: oh got it, sorry was mixed up.

**Pull request merged**